java/core/src/main/java/sleeper/core/properties/instance/BulkExportProperty.java

```diff
@@ -43,6 +44,58 @@ public interface BulkExportProperty {
             .defaultValue("800")
             .propertyGroup(InstancePropertyGroup.BULK_EXPORT)
             .runCdkDeployWhenChanged(true).build();
+    UserDefinedInstanceProperty BULK_EXPORT_LEAF_PARTITION_LANGUAGE = Index
+            .propertyBuilder("sleeper.bulk.export.leaf.partition.language")
+            .description("The language to use for the leaf partition bulk export processor.")
+            .defaultValue("java")
+            .validationPredicate(SleeperPropertyValueUtils::isValidBulkExportLanguage)
+            .propertyGroup(InstancePropertyGroup.BULK_EXPORT)
+            .runCdkDeployWhenChanged(true).build();
```

Comment:
It doesn't look like this is used? If we want this to use the same runner as compaction, we'll need to match `sleeper.table.compaction.method` and `sleeper.default.table.compaction.method`, which use an enum `CompactionMethod` for which runner should be used.

Can we leave this out for now?